crates/wasmtime/src/runtime/vm/libcalls.rs

```diff
@@ -109,6 +109,7 @@ pub mod raw {
                 ) $( -> libcall!(@ty $result))? {
                     $(#[cfg($attr)])?
                     {
+                        #[allow(deprecated)] // FIXME: need to update this
```

Comment:
Still a FIXME here and below in same file

Reply:
Ah yeah this is intentional where it's deferred to a future PR to handle the usages for libcalls